client/state/sites/selectors/get-woo-express-trial-days-left.ts

```diff
@@ -0,0 +1,20 @@
+import { AppState } from 'calypso/types';
+import getWooExpressTrialExpiration from './get-woo-express-trial-expiration';
+
+/**
+ * Get the number of days left in the Woo Express trial. If the trial is not active, returns null.
+ *
+ * @param {AppState} state - Global state tree
+ * @returns {null|number} Number of days left in the trial, or null if the trial is not active.
+ */
+export default function getWooExpressTrialDaysLeft( state: AppState ): number | null {
```

Comment:
General naming thought: it may be more future-proof to use `EcommerceTrial` instead of `WooExpressTrial` in all of these functions, as that gets us away from the branding and closer to the underlying entities. That would also align more clearly with the `ECOMMERCE_TRIAL` naming we are using elsewhere.